front/app/components/admin/PostManager/components/FilterSidebar/FilterRadioButton/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { Box } from '@citizenlab/cl2-component-library';
+import { hideVisually } from 'polished';
+import styled from 'styled-components';
+
+const HiddenRadio = styled.input.attrs({ type: 'radio' })`
+  ${hideVisually()};
+`;
+
+const Label = styled.label<{ selected: boolean }>`
+  display: flex;
+  align-items: center;
+  padding: 8px;
+  cursor: pointer;
+  border-radius: ${({ theme }) => theme.borderRadius};
+  width: 100%;
+  font-size: ${({ theme }) => theme.fontSizes.s}px;
+  color: ${({ theme: { colors } }) => colors.blue500};
+
+  &:hover {
+    background: ${({ theme: { colors } }) => colors.grey200};
+  }
+
+  background: ${({ theme: { colors }, selected }) =>
+    selected ? colors.background : colors.white};
+
+  ${HiddenRadio}.focus-visible + & {
+    outline: 2px solid black;
+  }
+`;
```

Comment:
https://www.notion.so/govocal/Add-BoxProps-to-Label-1629663b7b2680dca934c5e48504daca?pvs=4